app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+from app.players.player import Player
+
+
+def calculate_team_total_rating(team: list[Player]) -> int:
+    return sum([hero.get_rating() for hero in team])
```

Comment:
Do not use list comprehension inside of `sum` as mentioned [in the checklist](https://github.com/vvg04021968/py-elves-and-dwarves/blob/develop/checklist.md#code-efficiency).